### DISCUSSION TITLE
Logs

### DISCUSSION THREAD

**Discussion by johncadengo** [heyform/heyform#71] [Ideas]
Is there a way to show error logs in the self-hosted version? 

- In JS it looks like the code is obfuscated. Is there a way to have debug logs mode enabled?
- In docker logs, I don't see any errors when I encountered an error in the application. I'm assuming it's front-end only

**Comment by iMuFeng** [comment]
1. There is no obfuscation. [Decorators](https://www.typescriptlang.org/docs/handbook/decorators.html) from TS like [@Controller()](https://docs.nestjs.com/controllers) and [@Query()](https://docs.nestjs.com/controllers#request-payloads) are converted into JS syntax. After the conversion, corresponding sourcemap files are also generated so that error messages can pinpoint the exact line in TS where the error occurred.

![image](https://github.com/heyform/heyform/assets/1407009/346d842d-0257-4c3e-92a5-672368ba0941)

2. I just added a filter to capture global errors (bd9cfff)

![image](https://github.com/heyform/heyform/assets/1407009/20d3c657-dd48-4db0-adad-c93625bfd49f)
